packages/warehouses/src/warehouseClients/DatabricksWarehouseClient.ts

```diff
@@ -360,4 +360,46 @@ export class DatabricksWarehouseClient extends WarehouseBaseClient<CreateDatabri
                 return super.getMetricSql(sql, metric);
         }
     }
+
+    async getTables(
```

Comment:
Because I can't test this on databrics or trino, I've just copied the default method from `postgres`. I could also move this to the `BaseWarehouseClient` and not override this function here